### PR TITLE
[MM-32794] Reporting Sub Section

### PR DESCRIPTION
**Pull request by hahmadia**
#### Summary

Add Reporting SubSections

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-32794

#### Release Note

```release-note
None
```

### CODE REVIEW

**Review by BenCookie95 — APPROVED** [review]
Code LGTM!

File: api4/system.go [hunk 1]
```diff
@@ -386,12 +386,8 @@ func getAnalytics(c *Context, w http.ResponseWriter, r *http.Request) {
 		name = "standard"
 	}
 
-	permissions := []*model.Permission{
-		model.PERMISSION_SYSCONSOLE_READ_REPORTING,
-		model.PERMISSION_SYSCONSOLE_READ_USERMANAGEMENT_USERS,
```
Comment (sbishel) [inline api4/system.go:391]: Are we sure this permission check can be dropped?

**Review by sbishel — APPROVED** [review]

File: api4/system.go (hunk 1 above)
Comment (sbishel) [reply api4/system.go:391]: Nevermind, I see you added it to UserManager.

**Comment by furqanmlk** [comment]
Hi @hahmadia ,

Can you please resolve the conflicts ?

**Review by furqanmlk — APPROVED** [review]
Feature looks good, all subsection permission working fine

**Pull request merged**